.github/workflows/test.yml

```diff
@@ -0,0 +1,135 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - '*'
+      - '*/*'
+      - '**'
+      - '!gh-pages'
+  pull_request:
+    branches:
+      - '*'
+      - '*/*'
+      - '**'
+      - '!gh-pages'
+  workflow_dispatch:
+
+jobs:
```

Comment:
Please use a test matrix like this:

https://github.com/RDFLib/rdflib/blob/6aefe60ecd1f4c0b069919d898b3ed42a2d9cd03/.github/workflows/validate.yaml#L19-L27

Reply:
Parallel execution using the build matrix will result in concurrent API execution, which I think may be limited depending on the endpoint, so I'm using sequential execution.

Reply:
You can use this to control it: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#concurrency

Reply:
See:
https://github.com/Animenosekai/translate/pull/27

Reply:
I set `max-parallel: 1`. I cannot undertand how to do sequencial execution with `concurrency`.